packages/shared-business/src/components/SupportingDocumentCollection.tsx

```diff
@@ -384,6 +435,15 @@ export const SupportingDocumentCollectionWithRef = <Purpose extends string>(
           </LakeButton>
         </LakeButtonGroup>
       </LakeModal>
+
+      <LakeModal
+        visible={isNotNullish(currentMetadata)}
+        title={t("supportingDocuments.informations")}
+        icon="document-regular"
+        onPressClose={() => setCurrentMetadata(undefined)}
+      >
+        {currentMetadata?.values.map(data => <CopiableLine label={data.title} text={data.value} />)}
```

Comment:
let's make the behavior customizable, not all of the lines should be Copiable, could be a `type` in the metadata configuration (e.g. `text`, `copy`, …)

also let's wrap those in a `ReadOnlyFieldList` to be consistent with the rest